App.tsx

```diff
@@ -1,9 +1,50 @@
 import React from 'react';
-
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import MainScreen from './src/screens/MainScreen';
+import PlateInventoryScreen from './src/screens/PlateInventoryScreen';
+import SettingScreen from './src/screens/SettingScreen';
+
+const BottomTabNavigator = createBottomTabNavigator();
+
+//TODO:- Icons, customize to black
+//https://stackoverflow.com/questions/59304281/create-custom-bottom-tab-navigator-in-react-native
+
+function Tabs() {
+  return (
+    <BottomTabNavigator.Navigator
+      tabBarOptions={{
+        keyboardHidesTabBar: true,
+      }}
+    >
+      <BottomTabNavigator.Screen
+        name="Main"
+        component={MainScreen}
+        options={{
+          tabBarLabel: 'Home',
+        }}
+      />
+      <BottomTabNavigator.Screen
+        name="Inventory"
+        component={PlateInventoryScreen}
+      />
+      <BottomTabNavigator.Screen name="Settings" component={SettingScreen} />
+    </BottomTabNavigator.Navigator>
+  );
+}
 
-const AppContainer = () => {
-  return <MainScreen />;
-};
+export default function App() {
+  return (
+    <NavigationContainer>
+      <Tabs />
+    </NavigationContainer>
+  );
+}
 
-export default AppContainer;
+/**
```

Comment:
Oh, no need to leave a comment about the dependency here. You can note it in the pull request next time.

Plus the dependency will be added to the package.json and all I have to do is npm install.